hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -1240,6 +1240,15 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.namenode.access-control-enforcer-reporting-threshold-ms</name>
```

Comment:
Please move this right below `dfs.namenode.read-lock-reporting-threshold-ms` since they are related.